backend/src/main.rs

```diff
@@ -1,11 +1,111 @@
-#[macro_use] extern crate rocket;
+#[macro_use]
+extern crate rocket;
+
+use dotenv::dotenv;
+
+use rocket::response::status::Created;
+use rocket::serde::{json::Json, Deserialize, Serialize};
+use rocket::{fairing, fairing::AdHoc, Build, Rocket};
+
+use rocket_db_pools::{sqlx, Connection, Database};
+
+use sqlx::FromRow;
+
+#[derive(Database)]
+#[database("fluke")]
+struct Fluke(sqlx::PgPool);
+
+#[derive(Debug, Clone, Deserialize, Serialize, FromRow)]
+struct Message {
+    message: String,
+}
+
+#[derive(Debug, Clone, Deserialize, Serialize, FromRow)]
+struct MessageWithId {
+    id: i64,
+    message: String,
+}
```

Comment:
I hate, hate, hate, hate having 2 types here.  These should be combined, however I couldn't find a good patter for getting the same struct without an ID to work for creating AND retrieving.  The ID does not need to be present on creation since the DB does it for us, but having it in the struct, will result in a 400 error from the serializing validation on the `POST messages/` endpoint